speculos/mcu/vnc.py

```diff
@@ -60,15 +69,14 @@ def redraw(self, pixels):
         self.p.stdin.write(buf)
         self.p.stdin.flush()
 
-    def can_read(self, s, screen):
+    def can_read(self, s: int, screen: DisplayNotifier):
```

Comment:
what is `s` ? Replace with a meaningful value

Reply:
Replaced with `fd`. But I think it can be removed completely. Will propose a dedicated commit with this simplification.